editor/import/resource_importer_scene.cpp

```diff
@@ -46,6 +46,8 @@
 #include "scene/resources/sphere_shape_3d.h"
 #include "scene/resources/world_margin_shape_3d.h"
 
+#include <cctype>
```

Comment:
How do you use cctype? Is there a string function that can be reused?

Reply:
This needed for [isalpha](https://en.cppreference.com/w/cpp/string/byte/isalpha). I looked in the ustring class but didn't find an alternative. I can provide it by myself as it was done in [this](https://github.com/godotengine/godot/pull/43563/files) PR.

Reply:
I don't know if doing a regex search for "a-z" and "A-Z" is more readable or not.

Reply:
> In locales other than "C", an alphabetic character is a character for which std::isupper() or std::islower() returns non-zero or any other character considered alphabetic by the locale.

This is also complex.

Reply:
> I don't know if doing a regex search for "a-z" and "A-Z" is more readable or not.

This will not work correctly locales other than english. Is it okay? String represents a Unicode characters.

I also made a mistake. We should use [iswalpha](https://en.cppreference.com/w/cpp/string/wide/iswalpha) instead of `isalpha` for this case. But let me know if I should use something else.